index.html

```diff
@@ -4,484 +4,528 @@
 
-              <div class="right-wrapper__flying-icon slack-icon">
-                <img
-                  class="slack-img"
-                  src="./src/icons/grommet-icons_slack.svg"
-                  alt="slack icon"
+                <input
+                  class="location-input"
+                  type="text"
+                  placeholder="Location"
+                  name="text"
+                  value=""
                 />
               </div>
 
-              <!-- First flying card -->
-              <div class="right-wrapper__flying-card first-flying-card">
-                <div class="flying-card__img-block first-img">
-                  <div class="flying-card__circle">
-                    <img src="./src/icons/star.svg" alt="star" />
-                    <p class="flying-card__rating">4.8</p>
-                  </div>
-                </div>
-                <h2 class="flying-card__title">Maliha</h2>
-                <p class="flying-card__subtitle">UX Designer</p>
+              <button type="submit" class="button search-button" href="#">
+                Search
+              </button>
+            </form>
+          </div>
+        </div>
+
+        <!-- Graphics -->
+        <div class="graphics">
+          <img
+            class="graphics__img"
+            src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
+            alt="A satisfied lovely woman holds a modern cell phone"
+          />
+
+          <!-- Big ellipse -->
+          <img
+            class="graphics__ellipse-big"
+            src="./src/img/ellipse-big.svg"
+            alt="big ellipse"
+          />
+          <!-- Small ellipse -->
+          <img
+            class="graphics__ellipse-small"
+            src="./src/img/ellipse-small.svg"
+            alt="small ellipse"
+          />
+
+          <!-- Flying icons -->
+          <div class="graphics__flying-icon google-icon">
+            <img
+              class="google-img"
+              src="./src/icons/flat-color-icons_google.svg"
+              alt="google icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon paypal-icon">
+            <img
+              class="paypal-img"
+              src="./src/icons/simple-icons_paypal.svg"
+              alt="paypal icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon indeed-icon">
+            <img
+              class="indeed-img"
+              src="./src/icons/simple-icons_indeed.svg"
+              alt="indeed icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon slack-icon">
+            <img
+              class="slack-img"
+              src="./src/icons/grommet-icons_slack.svg"
+              alt="slack icon"
+            />
+          </div>
+
+          <!-- First flying card -->
+          <div class="graphics__flying-card first-flying-card">
+            <div class="flying-card__img-block first-img">
```

Comment:
```suggestion
            <div class="floating-card__img">
```

You may reach this block to apply individual styles by writing in CSS such selector: `.floating-card_purple .floating-card__img {}`.

Please, apply this comment to the card below too.